lib/SPIRV/SPIRVLowerLLVMIntrinsic.cpp

```diff
@@ -52,52 +52,66 @@ namespace SPIRV {
 namespace {
 
 typedef struct {
+  // Extension that is required for an emulation
+  const ExtensionID RequiredExtension;
   // Extension that supports the LLVM Intrinsic.
   // Thus, emulation is not needed if extension is enabled.
-  const ExtensionID SupportingExtension;
-  const char *SPIRVFuncName;
+  const ExtensionID ForbiddenExtension;
```

Comment:
It would be good to document how these interact, such as what happens if both extensions are enabled.

(I've tried thinking of a better name for "ForbiddenExtension", but couldn't yet).

Reply:
Added comment explaining how RequiredExtension and ForbiddenExtension interact.

Reply:
Thanks, that makes it clear enough I think.